fdbserver/MutationTracking.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * MutationTracking.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _FDBSERVER_MUTATIONTRACKING_H_
+#define _FDBSERVER_MUTATIONTRACKING_H_
+#pragma once
+
+#include "fdbclient/FDBTypes.h"
+#include "fdbclient/CommitTransaction.h"
+
+#define MUTATION_TRACKING_ENABLED 0
```

Comment:
Maybe moving this to the .cpp file is better, because then all modifications are just in one file and only one file needs to be recompiled.

Reply:
In my proposed solutions for getting rid of the overhead from constructing a trace event and making `detail()` calls on it when mutation tracking is disabled, this macro would have to stay in the header.